evm

```diff
@@ -118,23 +118,79 @@ function evm_system() {
   export PATH
 }
 
-function _unzip-strip() {
+function _unzip_strip() {
```

Comment:
If I get what you are trying to fix, this is the only thing that would be required. If that is so, please open 2 separated PRs. 1 to fix the error you are getting and another proposing a refactor.

Reply:
It takes me a lot of time, to switch cassette to make this requests, to make another one for this part. That I will not do. I think you can live to pushing this in this branch. For future reference do not name add "-" inside names.